packages/grid_client/scripts/vm_with_qsfs.ts

```diff
@@ -114,7 +115,7 @@ async function main() {
         cpu: 1,
         memory: 1024,
         rootfs_size: 0,
-        flist: "https://hub.grid.tf/tf-official-apps/base:latest.flist",
+        flist: flists.microVMs.find(flist => flist.name == "Ubuntu-24.04")!.flist,
```

Comment:
why do we need to use find? why can't we just use a data structure as shown below so that we can access an ubuntu-24.04 flist like this ```flist: fullVMs.Ubuntu["24.04"].value ``` as done in the other apps?

Reply:
It's the same, I propose to have the latest version object and we could update it from one place. 
@AhmedHanafy725 ??